### ISSUE TITLE
Classification using both ML algorithm and Deeplearning (DNN). #GSSoC22

### ISSUE THREAD

**Issue by AdityaTheDev** [srimani-programmer/Breast-Cancer-Predictor#180]
Im a Gssoc22 contributor

**Comment by freny24** [comment]
@AdityaTheDev Do give a proper description for the issue , how would you work on it?

**Comment by ApoorvYdv** [comment]
This is Apoorv Yadav
I would like to work on this issue
For the ML model i will be using classification algorithms like Logistic regression, svm and many more.
For the deep learning part i will be using Neural network and also apply transfer learning to get pretrained weights.
In the end i will be concluding with which has the best accuracy.

**Comment by AdityaTheDev** [comment]
> @AdityaTheDev Do give a proper description for the issue , how would you work on it?

I would like to create a neural network model for prediction. A basic Deep neural network not a CNN

**Comment by freny24** [comment]
@Apoorv-17 Hey! Applying transfer learning issue is already taken up  by a participant.

**Comment by freny24** [comment]
@AdityaTheDev could you please elaborate ? As RNN / CNN / LSTM / BRNN is already covered up in another issues!

**Comment by AdityaTheDev** [comment]
> @AdityaTheDev could you please elaborate ? As RNN / CNN / LSTM / BRNN is already covered up in another issues!

Sentiment analysis using various NLP techniques like rnn, birnn, lstm and brnn

**Comment by freny24** [comment]
@AdityaTheDev  Do work on it! compare different models in your code!
